### PR TITLE
Updated dependency gem to latest vesrsion

### PR DESCRIPTION
**Pull request by sunil-lakshman**
Upgraded activesupport gem to latest version. Rails 7 requires activesupport
 V7.0 and greater.

### CODE REVIEW

File: contentstack.gemspec [hunk 1]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
   s.files = `git ls-files`.split("\n")
   s.require_paths = ["lib"]
 
-  s.add_dependency 'activesupport', '>= 3.2', '< 6.2'
+  s.add_dependency 'activesupport', '>= 3.2', '< 7.0.4.2'
```
Comment (uttamukkoji) [inline contentstack.gemspec:23]: @sunil-lakshman Please check the version here: should be 7.0.4

File: contentstack.gemspec [hunk 2]
```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |s|
...
   s.add_dependency "contentstack_utils", '~> 1.0', '>= 1.0.1'
```
Comment (uttamukkoji) [inline contentstack.gemspec:24]: update to latest version

Comment (sunil-lakshman) [reply contentstack.gemspec]: Done

**Review by uttamukkoji — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,28 @@
 ## CHANGELOG
+
```
Comment (uttamukkoji) [inline CHANGELOG.md:2]: @sunil-lakshman add latest version `0.6.2` change log as well.

Comment (sunil-lakshman) [reply CHANGELOG.md:2]: Done

**Review by uttamukkoji — APPROVED** [review]

**Review by aravindbuilt — APPROVED** [review]

**Pull request merged**